docs/source/subgraphs.md

```diff
@@ -0,0 +1,126 @@
+---
+title: 'Working with Subgraphs'
+sidebar_title: 'Working with Subgraphs'
+description: 'How to use Rover to work with federated graphs and their implementing services'
+---
+
+Rover is Apollo's preferred way of working with federated graphs, and provides
+a number of useful utilities for doing just that.
+
+> Everything on this page applies only to federated graphs and their
+> implementing services. For a guide on how to work with a non-federated graph,
+> check out [Working with Graphs](./graphs).
+
+## Introspecting Implementing Services
+
+Implementing services (subgraphs) in a federated graph have additional
+information that can't be expressed in a normal introspection request like
+`@key` values, `@external` definitions, etc. There is a special field available
+in subgraphs implementing the apollo federation spec, which can be used to get
+the full schema of a subgraph.
+
+Rover's `subgraph introspect` command makes fetching SDL from a running subgraph
+easier than ever:
+
+```bash
+rover subgraph introspect http://localhost:4001
+```
+
+This command outputs the "special" introspection result to `stdout`, which can
+be piped to other commands or output to a file.
+
+> For more on how `stdout` works, see [Essential Concepts](./essentials#using-stdout).
+
+> TODO: headers API
+
+## Pushing a subgraph to Apollo Studio
+
+> All of the following examples require you to first create an account with
+> Apollo Studio, [authenticate Rover]
+> (./configuring#authenticating-with-apollo-studio) and create a new graph to
+> publish to.
+
+Apollo Studio is a cloud platform that helps you build, validate, and secure
+your organization's data graph. The Apollo schema registry, as part of Apollo
+Studio is a free tool to allow you to track schema versions and validate
+changes, all while acting as the single source of truth for the question of
+"what schema is the correct one?"
+
+For federated graphs, Apollo Studio also plays the key role in composing the
+subgraphs to create the final schema.
```

Comment:
this seems to be copy pasted directly from the `graph` page. I'm wondering if it may be worth it to not duplicate this section somehow...

Reply:
I shortened this a bit and added a link to the docs for apollo studio for more info!